synopsys-task/src/synopsys-task/synopsys-bridge.ts

```diff
@@ -469,4 +522,45 @@ export class SynopsysBridge {
     }
     return bridgeDownloadUrl;
   }
+
+  getLatestVersionUrl(): string {
+    const osName = process.platform;
+    let bridgeDownloadUrl = this.bridgeUrlLatestPattern;
+    if (osName === "darwin") {
+      bridgeDownloadUrl = bridgeDownloadUrl.replace(
+        "$platform",
+        this.MAC_PLATFORM
+      );
+    } else if (osName === "linux") {
+      bridgeDownloadUrl = bridgeDownloadUrl.replace(
+        "$platform",
+        this.LINUX_PLATFORM
+      );
+    } else if (osName === "win32") {
+      bridgeDownloadUrl = bridgeDownloadUrl.replace(
+        "$platform",
+        this.WINDOWS_PLATFORM
+      );
+    }
+
+    return bridgeDownloadUrl;
+  }
+
+  async setBridgeExecutablePath(
+    osName: string,
+    filePath: string
+  ): Promise<string> {
+    if (osName === "win32") {
+      this.bridgeExecutablePath = path.join(
+        filePath,
+        constants.SYNOPSYS_BRIDGE_EXECUTABLE_WINDOWS
+      );
+    } else {
```

Comment:
use else if and check for linux and mac OS only, since we are not supporting any other OS

Reply:
we can use the code, no need to change.